lib/turbo_tests/windows.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module TurboTests
+  class Runner
+    private
+
+    def start_subprocess(env, extra_args, tests, process_id, record_runtime:)
+      if tests.empty?
+        @messages << {
+          type: "exit",
+          process_id: process_id,
+        }
+      else
+        require "securerandom"
+        env["RSPEC_FORMATTER_OUTPUT_ID"] = SecureRandom.uuid
```

Comment:
Let's just replace `start_subprocess` in `lib/turbo_tests/runner.rb` so code for all platforms is the same.